src/addresses.ts

```diff
@@ -147,6 +156,7 @@ const BASE_GOERLI_ADDRESSES: ChainAddresses = {
   v3CoreFactoryAddress: '0x9323c1d6D800ed51Bd7C6B216cfBec678B7d0BC2',
   multicallAddress: '0xB206027a9E0E13F05eBEFa5D2402Bab3eA716439',
   quoterAddress: '0xedf539058e28E5937dAef3f69cEd0b25fbE66Ae9',
+  quoterV2Address: '0xedf539058e28E5937dAef3f69cEd0b25fbE66Ae9',
```

Comment:
On the testnets and Base, I believe Quoter isn't deployed (only QuoterV2). I kept the quoter address the same (V1) and added a V2 with the same value.